CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Platform 
 The Tidepool platform API
 
+## Unreleased
+### Changed
```

Comment:
Maybe the Tidepool changelog below is no longer relevant ?
Since they have stopped to update it

Reply:
I don't understand this point...

Reply:
There is a long list of changes made by tidepool, below our changelog.
But this changelist seems not being maintain anymore.

Reply:
Yes but that's the point of a changelog keeping history of what's been done... So why remove it now ?